superset-frontend/src/components/Tabs/Tabs.tsx

```diff
@@ -119,6 +119,9 @@ const StyledEditableTabs = styled(StyledTabs)`
     `}
 `;
 
+const StyledCancelXIcon = styled(Icons.CancelX)`
+  color: ${({ theme }) => theme.colors.grayscale.base};
```

Comment:
Same comment as in another PR. Here we should define whether the standard is to use the `iconColor` prop or go with this approach. Additionally, we might want to use this color as the base for the Icons component in general, to avoid repetition.